source/includes/read/change-streams.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
```

Comment:
In all of the original PHPLIB code examples, we fall back to `mongodb://127.0.0.1/` (see: [aggregate.php](https://github.com/mongodb/mongo-php-library/blob/1.19.1/examples/aggregate.php#L23)). Why not do so here?

Reply:
The message about setting the MONGODB_URI variable was a request from Jerome on a previous PR, but I'm fine with either option. If you have a strong preference for `mongodb://127.0.0.1/` I'm happy to change it

Reply:
Happy to leave as-is if this @GromNaN already signed off on this approach.

Reply:
This examples are using the [sample data](https://www.mongodb.com/docs/atlas/sample-data/) from Atlas. Do you know if their is any documented way to get this data locally?

Reply:
Sorry, I confused these with the data sets we already discuss in the PHPLIB docs: https://www.mongodb.com/docs/php-library/current/tutorial/example-data/

I poked around https://www.mongodb.com/docs/atlas/sample-data/ and it didn't see any way to download those directly :disappointed: